UI/accounts/edit.html

```diff
@@ -257,6 +257,22 @@
               value = '1'} ?>
    </div>
 </div>
+<?lsmb IF form.custom_link_descriptions.size ?>
+<div class="inputline" id="acc-custom-flags-line">
```

Comment:
does this imply that there are only ever going to be as many checkmarks as fit on a single row? (How many checkmarks are there on the other rows? 6?) Or should we support wrapping to new rows by limiting the number on the row to the same number on the "built-in" rows?

Reply:
It's a good point, but as things stand, it will wrap, albeit not very prettily.
I think this is a CSS concern.
We can't wrap based purely on the number of checkmarks as (unlike the other fixed checkmarks) we have no control over how long the label is for each.

Reply:
Ok. Good point. Other than that, I think this is a nice and clean PR. Thanks!